File: algorithms/bubble-sort/bubble_sort.cs
using System;

public void BubbleSort(int[] array) {
  int i,j;
  int cache;

  for(i = 0; i < array.Length; i++) {
    for(j = 0; j < array.Length; j++) {
      if (array[j] > array[j + 1]) {
        cache = array[j];
        array[j] = array[j + 1];
        array[j + 1] = cache;
      }
    }
  }

  Console.WriteLine(array);
}

File: algorithms/bubble-sort/bubble-sort.js
const bubbleSort = (arr) => {
  for(let i = 0; i < arr.length; i++) {
    for(let j = 0; j < arr.length; j++) {
      if (arr[j] > arr[j + 1]) {
        const cache = arr[j]
        arr[j] = arr[j + 1]  
        arr[j + 1] = cache
      }
    }
  }

  console.log(arr)
}

console.log(bubbleSort([3,4,1,7,2,9]))
